### PR TITLE
Format Code and Fix Lints with GDScript-Toolkit

### PR DESCRIPTION
**Pull request by lishaduck**
The code changes in this pull request include formatting the code using gdformat and fixing linting issues. These changes improve the readability and maintainability of the codebase.

* format with gdformat
* fix gdlint lints

### CODE REVIEW

**Comment by lishaduck** [comment]
@MattsAttack, you see if this is all makes sense? I think its relatively straightforward, but I want to make sure this isn't a big disruption before I merge it.

File: number_balls/number_two.gd [hunk 1]
```diff
@@ -2,10 +2,11 @@ extends Area2D
 
 var travelled_distance = 0
 
-var num = 2
+const num = 2
+
 
 ## Called when the node enters the scene tree for the first time.
```
Comment (lishaduck) [inline number_balls/number_two.gd:8]: This comment is outdated, but that can be fixed later.
```suggestion
```

File: rock.gd [hunk 2]
```diff
@@ -1,7 +1,9 @@
 extends StaticBody2D
 
-@export var answer = 1
+@export var answer: int = 1
+
+
 # Called when the node enters the scene tree for the first time.
```
Comment (lishaduck) [inline rock.gd:6]: This comment is outdated, but that can be fixed later.
```suggestion
```

File: number_balls/number_one.gd [hunk 3]
```diff
@@ -2,10 +2,11 @@ extends Area2D
 
 var travelled_distance = 0
 
-var num = 1
+const num = 1
+
 
 ## Called when the node enters the scene tree for the first time.
```
Comment (lishaduck) [inline number_balls/number_one.gd]: This comment is outdated, but that can be fixed later.
```suggestion
```

File: number_balls/number_two.gd (hunk 1 above)
Comment (lishaduck) [inline number_balls/number_two.gd]: This comment is outdated, but that can be fixed later.
```suggestion
```

File: rock.gd (hunk 2 above)
Comment (lishaduck) [inline rock.gd]: This comment is outdated, but that can be fixed later.
```suggestion
```

**Review by lishaduck — COMMENTED** [review]
Found some outdated comments. I might as well remove them.

**Comment by MattsAttack** [comment]
All of it but the : before the =. I understand the ones after the variable names cause you specify data types that way. But the : before the = signs are kind of wonky

**Review by MattsAttack — APPROVED** [review]

**Comment by lishaduck** [comment]
> All of it but the : before the =. I understand the ones after the variable names cause you specify data types that way. But the : before the = signs are kind of wonky

Yeah. They repurposed the walrus operator and I'm torn over weather it's evil or genius. `:=` is basically `:<type> =`, which makes sense, but it isn't how Python does it.

**Pull request merged**